dags/transportation/dockless/create_la_boundary.sql

```diff
@@ -0,0 +1,12 @@
+    
```

Comment:
Is this used anywhere at the moment, or do you intend to use it somewhere?

Reply:
yeah, this function is used by folks when querying